middleware/csrf_test.go

```diff
@@ -0,0 +1,70 @@
+package middleware_test
+
+import (
+	"testing"
+
+	"github.com/gobuffalo/buffalo"
+	"github.com/gobuffalo/buffalo/middleware"
+	"github.com/gobuffalo/buffalo/render"
+	"github.com/markbates/willie"
+	"github.com/stretchr/testify/require"
+)
+
+type csrfForm struct {
+	AuthenticityToken string `form:"authenticity_token"`
+}
+
+func ctCSRFApp() *buffalo.App {
+	h := func(c buffalo.Context) error {
+		return c.Render(200, render.String(c.(*buffalo.DefaultContext).Data()["authenticity_token"].(string)))
```

Comment:
`return c.Render(200, render.String(c.Value("authenticity_token").(string)))` - it's a lot shorter and easier on the eyes. :)